libs/src/sdk/api/generated/default/apis/PlaylistsApi.ts

```diff
@@ -63,6 +73,26 @@ export interface SearchPlaylistsRequest {
  */
 export class PlaylistsApi extends runtime.BaseAPI {
 
+    /**
+     * Check if Playlist ID is occupied
+     */
+    async checkIfPlaylistIDIsOccupied(requestParameters: CheckIfPlaylistIDIsOccupiedRequest): Promise<NonNullable<OccupiedResponse["data"]>> {
```

Comment:
we don't necessarily need this API anymore since we're just going with the random ID